### PR TITLE
add template %(module_name_full)s

### PR DESCRIPTION
**Pull request by smoors**
for background, see discussion here: https://github.com/easybuilders/easybuild-easyblocks/pull/1857/files

### CODE REVIEW

**Comment by smoors** [comment]
for some reason travis is still indicated as pending even though all its tests have passed.
will close and reopen..

File: easybuild/framework/easyblock.py [hunk 1]
```diff
@@ -2799,7 +2799,10 @@ def update_config_template_run_step(self):
         """Update the the easyconfig template dictionary with easyconfig.TEMPLATE_NAMES_EASYBLOCK_RUN_STEP names"""
 
         for name in TEMPLATE_NAMES_EASYBLOCK_RUN_STEP:
-            self.cfg.template_values[name[0]] = str(getattr(self, name[0], None))
+            if name[0] == 'module_name_full':
+                self.cfg.template_values[name[0]] = ActiveMNS().det_full_module_name(self.cfg)
```
Comment (boegel) [inline easybuild/framework/easyblock.py]: You should be able to use `self.cfg.full_mod_name` here (no need to re-determine the module name)

File: easybuild/framework/easyconfig/templates.py [hunk 2]
```diff
@@ -73,6 +73,7 @@
 TEMPLATE_NAMES_EASYBLOCK_RUN_STEP = [
     ('installdir', "Installation directory"),
     ('builddir', "Build directory"),
+    ('module_name_full', "Full module name"),
```
Comment (boegel) [inline easybuild/framework/easyconfig/templates.py]: wouldn't `full_module_name` make more sense?

**Comment by smoors** [comment]
> @smoors I think you actually want/need the "short" module name for the SCOOP use case, which is the "user-facing" module name (for example `SCOOP/0.7.1.1` when using a module hierarchy, rather than `Compile/GCCcore/8.2.0/SCOOP/0.7.1.1`, which should never be loaded using that full name).

@boegel ok, I am not sure how HMNS works exactly. but in order to load SCOOP like that, don't you have to load the toolchain first to make it available?:
```
module load GCCcore/8.2.0 SCOOP/0.7.1.1
```

File: easybuild/framework/easyconfig/templates.py (hunk 2 above)
Comment (smoors) [reply easybuild/framework/easyconfig/templates.py]: depends how you look at it :) either way is fine to me.

**Comment by smoors** [comment]
this is going to be a pain to get right for every possible setup, so I will just define the module load command in the easyconfig instead.

**Pull request closed without merging**